src/index.ts

```diff
@@ -3,19 +3,21 @@ import { WinstonAdaptor } from 'typeorm-logger-adaptor/logger/winston';
 
 import logger from './logger';
 
-import { API, funnyHeaderMiddleware } from './common';
+import { API, funnyHeaderMiddleware, gameMiddleware } from './common';
 import { ReportController } from './report';
 import { AuthController } from './auth';
 import { UserController } from './user';
+import { GameController } from './game';
 import { FeedController } from './feed/feed.controller';
 
 const server = new API({
-    middlewares: [funnyHeaderMiddleware],
+    middlewares: [funnyHeaderMiddleware, gameMiddleware],
```

Comment:
We don't want to use this middleware globally. It will probably break other endpoints.